osu.Game/Graphics/Backgrounds/Triangles.cs

```diff
@@ -77,6 +77,12 @@ public float TriangleScale
             set => triangleScale.Value = value;
         }
 
+        /// <summary>
+        /// If enabled, only the portion of triangles that falls within this <see cref="Drawable"/>'s
+        /// shape is drawn to the screen.
+        /// </summary>
+        public bool Masking { get; set; } = true;
```

Comment:
Historically, we have avoided having properties like this with differing defaults. Would you be against having this `false` by default and setting it to `true` in all the usages that require it?

Reply:
Hmm. Now that I'm thinking about this and the reason I'm even trying to implement this - that sounds reasonble.
So the end goal is add ability to draw triangles within a drawable without the need of wrapping them into a container with masking. So far we don't have such cases, however I [stumbled upon one](https://github.com/ppy/osu/pull/21527#issuecomment-1340810903).

TL;DR I'm fine with that, sure. There are no cases in which changing masking to true will be required though since this component is being placed within masked containers in all current usages.